src/main/kotlin/com/intellij/plugin/gamification/config/Config.kt

```diff
@@ -0,0 +1,8 @@
+package com.intellij.plugin.gamification.config
+
+object Config {
+    object Dialog {
+        const val with = 400
```

Comment:
Since the constants are used only inside the single dialog it's better to hide them all as its static (companion object's) fields.

Reply:
Also, overriding `getDimensionServiceKey` method from `DialogWrapper` makes dialog to remember location and size of dialog between showings and IDE restarts.